dali/server/daserver.cpp

```diff
@@ -365,6 +367,14 @@ int main(int argc, char* argv[])
             port = atoi(argv[++i]);
         else if (streq(argv[i],"--rank") || streq(argv[i],"-r"))
             myrank = atoi(argv[++i]);
+#ifdef _DEBUG
+        else if (streq(argv[i],"--hold"))
```

Comment:
See comment below re hold mode

Reply:
I assume this will disappear once dali calls loadConfiguration?

Reply:
Yes